docs/upgrading.md

```diff
@@ -0,0 +1,66 @@
+# Upgrading Contour to 0.15
```

Comment:
Make this # Upgrading Contour 

(drop the version number)

Then add an introductory paragraph.

This document describes how to upgrade from Contour 0.14 to 0.15.